openmls/src/framing/plaintext.rs

```diff
@@ -115,6 +115,11 @@ impl MlsPlaintext {
         &self.signature
     }
 
+    #[cfg(test)]
+    pub fn test_signature(&self) -> &Signature {
```

Comment:
Note that all this will go away again because the signature doesn't exist anymore.

Reply:
Can you say more? I needed that for testing. What do you propose to do?

Reply:
`MLSPlaintext` won't have a signature anymore. You can get it from the `MLSContentAuthData`. It's just a note here. Nothing to do.